### PR TITLE
Updated IBM Resilient integration

### PR DESCRIPTION
**Pull request by stevengoossensB**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [] In Progress
- [ x ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
N/A

## Description
Limits the amount of incidents to fetch in one go.

## Minimum version of Cortex XSOAR
- [ x ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ x ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by thefrieddan1** [comment]
Hi @stevengoossensB, 
In order for me to start review your PR 
Please be sure to fill the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA).

I noticed that the version of pack need to be updated in pack_metadata.json
Please notice the next pack version should be 1.1.8 and not 1.0.12
You can use `demisto-sdk update-release-notes` 
see more on this here:
[demisto-sdk update-release-notes](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/update_release_notes/README.md)

Thanks

**Comment by thefrieddan1** [comment]
Hi @stevengoossensB,
A kind reminder in order for me to start review your PR
Please be sure to fill the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA).

File: Packs/IBMResilientSystems/ReleaseNotes/1_1_8.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### IBM Resilient Systems
+- Updated the Docker image to: *demisto/resilient:2.0.0.62392*.
+- Added a parameter max_fetch for the integration (default value=50) to control the maximum amount of incident to fetch in one go.
```
Comment (ShirleyDenkberg) [inline Packs/IBMResilientSystems/ReleaseNotes/1_1_8.md:5]:
```suggestion
- Added the *max_fetch* parameter (default value=50) to control the maximum amount of incidents to fetch at a time.
```

**Comment by ShirleyDenkberg** [comment]
@thefrieddan1 @GuyAfik @JudahSchwartz Doc review completed.

**Comment by thefrieddan1** [comment]
Hi @stevengoossensB, 
Lets please schedule a demo for the addition in order to proceed.
I have tried to contact you in the slack DFIR as well.

Thanks,

**Comment by thefrieddan1** [comment]
Hi @stevengoossensB 
Although we greatly appreciate your contribution, it has been rejected for the following reason: 
No reply.

Thank you for understanding.
We look forward to receiving more contributions from you in the future.

**Pull request closed without merging**